### DISCUSSION TITLE
Tauri app on win10 show error: cannot find vcruntime140.dll, cannot find vcruntime140_1.dll

### DISCUSSION THREAD

**Discussion by crackleeeessyp** [tauri-apps/tauri#11880] [Q&A]
I copied the release binaries from win11 to win10, when start tauri app, it shows error: cannot find vcruntime140.dll and vcruntime140_1.dll, I need to copy these two files to tauri exe folder to make it work. how can I solve this problem by embed these two dlls to exe?

**Comment by FabianLars** [comment]
you can bundle them as a `resource` but that requires the use of the installers and not just the exe. including the _dynamic_ libs inside the app exe is not possible. statically linking vcruntime should be possibly and is done by tauri by default which is why this issue surprises me tbh.

are you compiling with msvc or gnu?
do you have any noteworthy additional rust dependencies?

**Reply by crackleeeessyp** [reply]
Tried multiple times, and I found that seems only debug version has this problem, and release version does not throw error. So does tauri disable it for debug one?

**Reply by FabianLars** [reply]
oh yeah, that could be. The code responsible doesn't look like that to me but i may be missing something (i'm only on my phone rn)

**Reply by crackleeeessyp** [reply]
Thanks~ I can use release version first
